app/Resources/translations/messages.de.xliff

```diff
@@ -199,6 +199,18 @@
                 <source>post.no_posts_found</source>
                 <target>Keine Beiträge gefunden.</target>
             </trans-unit>
+            <trans-unit id="post.created_successfully">
+                <source>post.created_successfully</source>
+                <target>Beitrag erstellt!</target>
+            </trans-unit>
+            <trans-unit id="post.updated_successfully">
+                <source>post.updated_successfully</source>
+                <target>Beitrag bearbeitet!</target>
+            </trans-unit>
+            <trans-unit id="post.deleted_successfully">
+                <source>post.deleted_successfully</source>
+                <target>Beitrag gelöscht!</target>
```

Comment:
i think we should use the long version from @javiereguiluz 

```Der Beitrag wurde erfolgreich gelöscht!```

same for the both above

```Der Beitrag wurde erfolgreich bearbeitet!```
```Der Beitrag wurde erfolgreich erstellt!```